mimic/canned_responses/maas_alarm_examples.py

```diff
@@ -294,3 +294,20 @@ def alarm_examples():
             "type": "whole number (may be zero padded)"
         }]
     }]
+
+
+def alarm_examples_template(metric, crit_threshold, warn_threshold):
+    """
+    There's actually a POST version of this to ask for a specific alarm example
```

Comment:
This docstring doesn't really explain anything about what the function does.

We should start getting better about adding `:param:` and `:return:` declarations for what all of our functions take and return, and since I can't understand what this is doing, it seems like this would be a good place to start :).